### DISCUSSION TITLE
Feature request: option to close a PR as "Applied" (not merged but somehow accepted)

### DISCUSSION THREAD

**Discussion by tgross35** [community/community#59487] [Pull Requests] [answered]
### Select Topic Area

Product Feedback

### Body

It's not uncommon for a pull request to be accepted and somehow applied, but not merged directly. The only option a maintainer then has is to close the PR - but this can be confusing because it looks like the change was rejected (it gets the red icon) but it was actually applied and just not merged.

Some examples of when this happens:

1. A maintainer applies the patch themselves, possibly fixing merge conflicts or changing the target branch
2. Portions of the PR were cherry picked
3. The Github repo acts as a mirror, approved PRs get applied upstream via a mailing list
4. The commits were merged along with others in a rollup style

These situations are fairly common, having a way to make the circumstances more clear would significantly improve understandability.

**Comment by jmarshall** [comment, marked as answer]
See also #6414.
